### PR TITLE
added globalStyle and created HomePage

### PR DESCRIPTION
**Pull request by ValentinCop**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]

File: frontend/src/pages/Home/index.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <Style>
+      <section className="banner">
+        <img src={logo} className="logo" alt="logo-adan" />
+        <p>
+          ADAN, un outil simple, intuitif, qui garantit aux artistes et à leur
+          employeur une <br /> cadre sécurisé et légal dédié à la mise en place
+          de prestations
+        </p>
+      </section>
+      <section />
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/index.jsx]: Why?

File: frontend/src/pages/Home/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <section />
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/index.jsx]: Why?

File: frontend/index.html [hunk 3]
```diff
@@ -3,6 +3,12 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (LuigiMenez) [inline frontend/index.html]: Not import fonts in the index.html

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**